Application/EdFi.Ods.Standard/Standard/5.0.0/Artifacts/Metadata/ApiModel.json

```diff
@@ -1,5 +1,5 @@
 {
-  "odsApiVersion": "7.0",
+  "odsApiVersion": "7.1",
```

Comment:
Since 4.0.0 will run in 7.1, we need this change also in 4.0.0 folder

Reply:
I updated it